samples/kafka/python/producer.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+#
+#Copyright 2016 Confluent Inc.
+# 
+# Modified for use with Azure Event Hubs for Apache Kafka Ecosystems
+# 06/18/2018
+#
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+#
+
+#
+#Example Kafka Producer.
+#
+from confluent_kafka import Producer
+import sys
+
+if __name__ == '__main__':
+    if len(sys.argv) != 2:
+        sys.stderr.write('Usage: %s <topic>\n' % sys.argv[0])
+        sys.exit(1)
+    topic = sys.argv[1]
+
+    #Producer configuration
+    #See https: //github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md
+    #See https://github.com/edenhill/librdkafka/wiki/Using-SSL-with-librdkafka#prerequisites for SSL issues
+    conf = {
+        'bootstrap.servers': '{YOUR.EVENTHUB.FQDN}',
```

Comment:
>EVENTHUB [](start = 36, length = 8)

minor nitpick: Namespace might be a better name here (but the down side is that only Azure EventHub customer is well aware of what does "namespace" means)